README.md

```diff
@@ -1,2 +1,48 @@
 # mobile-view
-Personal portfolio mobile-view 
+
+This project is about creating a mobile view of my personal portifolio
+
+## Built with
+
+-HTML
+-CSS
+
+## setup to access the project
+
+> To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+-Github Account
+-code editor
+-Git
+
+### Setup
+
+(go to the terminal in the code editor and put the following git commands)
+-git init
+-git clone https://github.com/yusufnviiri/mobile-view.git
+-cd portifolio-mobile
+-open index.html in the editor/browser to view the project contents
+-open style.css to see the CSS styles used in the project.
+-enjoy
+
+## Author
+
+### :MAN: Yusuf Nviiri
+
+-GitHub: @yusufnviiri
+-Twitter:@YNviiri
+-LinkedIn: yusuf nviiri
+
```

Comment:
[ Required Changes ]

- [ ] Kindly add links to your social media accounts. Use this [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) format and place your account links so if someone clicks on any of your accounts, can then be taken to that specific account. 👍